synfig-studio/src/gui/canvasview.cpp

```diff
@@ -298,6 +302,21 @@ class studio::CanvasViewUIInterface : public UIInterface
 	virtual bool
 	amount_complete(int current, int total)
 	{
+		float temp = (float)current/(float)total;
+		if(temp != cur_progress && temp<1.0)
```

Comment:
In case if `temp == cur_progress` (progress didn't changed) I expect nothing to happen.

But with current code if `temp == cur_progress` then progressbar will be hidden (see code after condition block).